detox/src/devices/Device.js

```diff
@@ -123,6 +113,20 @@ class Device {
     return !params.delete && !params.newInstance && this._processes[_bundleId];
   }
 
+  _assertHasSingleParam(singleParams, params) {
```

Comment:
Isn't the "assert" moniker incorrect if it has the side-effect of returning the count? Should be "count" and the `throw` should be in the calling method.